tests/tap.sh

```diff
@@ -42,8 +42,8 @@ for ((i=0;i<${#backends[@]}; ++i)); do
         continue
     fi
 
-    # grep to skip test if backend cannot handle Blocked Restriction
-    if grep -F -q -e 'Backend does not support ElemRestrictionCreateBlocked' \
+    # grep to skip test if backend support a function
```

Comment:
Right now I changed tap to 'SKIP' whenever the error message contains 'Backend does not support' but I'd like to move to explicit whitelisting/blacklisting.

Reply:
Fair for now.

As for white/blacklists, one thought is that we could make a `CeedInitTest("resource", "list,of,components", &ceed)` (or encode requirements in the resource string) that fails immediately if required components are not implemented. Then a backend would declare what it supports up-front and tests would specify programmatically which interface levels are needed. So if a backend claims to support `CeedBasisApply` then we expect every test that uses it to pass. I don't know if that is better than listing backends in the test source files, but I'm concerned that such lists could become messy and we could end up skipping some tests that should not be.